### PR TITLE
build: Add workflow for users to self-assign issues by comment

### PR DESCRIPTION
**Pull request by sarina**

### CODE REVIEW

**Comment by sarina** [comment]
@Carlos-Muniz wanna take a look?

note I have this running & tested:

https://github.com/openedx/platform-roadmap/issues/128 as well as https://github.com/sarina/gh-scripting/issues/1 testing that an out-of-org member could use successfully

**Review by Carlos-Muniz — APPROVED** [review]
Hey it looks great. At the bottom of the each of the files there's a notification saying that there isn't a new line at the bottom of each files.

File: .github/workflows/self-assign-issue.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#  issue_comment:
+#    types: [created]
+jobs:
+  add_assignee:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions-ecosystem/action-add-assignees@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          assignees: ${{ github.triggering_actor }}
```
Comment (Carlos-Muniz) [inline .github/workflows/self-assign-issue.yml:20]:
```suggestion
          assignees: ${{ github.triggering_actor }}

```

File: workflow-templates/self-assign-issue.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+# ticket
+
+name: Add newly created GitHub Request tickets to the tCRIL Engineering project board
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  self_assign_by_comment:
+    uses: openedx/.github/.github/workflows/self-assign-issue.yml@master
```
Comment (Carlos-Muniz) [inline workflow-templates/self-assign-issue.yml:12]:
```suggestion
    uses: openedx/.github/.github/workflows/self-assign-issue.yml@master

```

**Review by nedbat — CHANGES_REQUESTED** [review]

**Review by nedbat — APPROVED** [review]

**Pull request merged**